Weaver/RealmWeaver.Fody/ModuleWeaver.cs

```diff
@@ -187,6 +187,8 @@ public void Execute()
             }
         }
 
+        LogDebug("trololo");
```

Comment:
Haha, yes - I was getting surprising results so wanted to confirm that the correct weaver was processing the assembly.